### PR TITLE
Setspeed indicator, larger hud size, more frame info

### PR DESCRIPTION
**Pull request by ecraven**
set-speed

### CODE REVIEW

**Comment by nozmajner** [comment]
Nice, it does feel better at this scale.
As @StanPancakes mentioned on IRC, the azimuthal radar shares the scale setting, which makes it quite big.
radartoobig

File: data/pigui/modules/radar.lua [hunk 1]
```diff
@@ -85,7 +85,7 @@ local function displayRadar()
 	if #radar > 0 then
 
 		local cntr = Vector(ui.screenWidth / 2, ui.screenHeight - ui.reticuleCircleRadius - 15)
-		local size = ui.reticuleCircleRadius
+		local size = ui.reticuleCircleRadius * 0.66
```
Comment (radcapricorn) [inline data/pigui/modules/radar.lua]: perhaps keep the 15 pixel offset fixed, i.e. `ui.screenHeight - size - 15`?

Comment (ecraven) [inline data/pigui/modules/radar.lua]: ah, thanks for the pointer, of course ;) I'll push that in a minute.

**Comment by ecraven** [comment]
set-speed

**Pull request merged**